governance/SIG Security Charter.md

```diff
@@ -1,97 +1,73 @@
 # SIG Security Charter
 
-This charter adheres to the Roles and Organization Management specified in <sig-governance>.
- Team information may be found in the <readme.md>
-
 ## Overview of SIG
-A central knowledge advisory group that provides resources and information security compliant, resolution and XYZ.
-
-All about security of systems and infrastructure resources. Security patching, resources and infra up to date.
-Provide the basic services of APPSEC related to this project.
-
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
-
-- Responsibility
+SIG Security is a special interest group for O3DE code security that works as a central knowledge advisory group and  provides resources, consultation, reporting mechanisms  and information for security compliance, and security related issue resolution. This SIG also helps in keeping security patching up to date, and provide the basic services related to  APPSEC (Application Security) of O3DE.
 
 ## Goals
 
-- Major goals that SIG seeks to generally achieve
+- To be set after rest of the charter is reviewed.
 
 ## Scope
```

Comment:
As worded, I don't really understand the point of this section in addition to the below "In scope" section. It also reads more like a "Goals" section, esp. the 2 bullets re: Identifying potential risks in the code and defining best practices. Can we remove it, or is it part of some standard charter template?

Reply:
Its part of the standard O3DE charter template.